Library/Formula/duck.rb

```diff
@@ -0,0 +1,29 @@
+class Duck < Formula
+  homepage "https://duck.sh/"
+  url "https://dist.duck.sh/duck-src-16336.tar.gz"
+  version "4.6.2.16336"
+  head "https://svn.cyberduck.io/trunk/"
+  sha1 "377f8ca3d111416a20122e0e5f41f189c5eda99e"
+
+  depends_on :java => ["1.7", :build]
+  depends_on :xcode => :build
+  depends_on "ant" => :build
+  depends_on "openssl"
+
+  def install
+    system "ant", "-Dbuild.compile.target=1.7", "-Drevision=16336", "cli"
```

Comment:
The `revision` is normally set from `svnversion` but because we build from a exported source tarball with no revision information it is set explicitly. We make a source tarball in our [continuous build](https://ci.cyberduck.io/job/terminal-release/).